### PR TITLE
Added Docker support

### PR DESCRIPTION
**Pull request by nklimovych**
- Added Docker dependency in pom.xml. 
 - Added Dockerfile, and docker-compose properties and added .env to gitignore.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,4 +30,7 @@ build/
 !**/src/test/**/build/
 
 ### VS Code ###
-.vscode/
\ No newline at end of file
+.vscode/
+
+### Docker ###
+/.env
```
Comment (vikkushnir) [inline .gitignore]:
```suggestion
/.env

```

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      - "${SPRING_LOCAL_PORT}:${SPRING_DOCKER_PORT}"
+      - "${DEBUG_PORT}:${DEBUG_PORT}"
+    environment:
+      SPRING_APPLICATION_JSON: '{
+      "spring.datasource.url": "jdbc:mysql://mysqldb:${MYSQL_DOCKER_PORT}/${MYSQL_DATABASE}?serverTimezone=UTC",
+      "spring.datasource.username": "${MYSQL_USER}", 
+      "spring.datasource.password": "${MYSQL_ROOT_PASSWORD}", 
+      "jwt.secret": "${JWT_SECRET}"
+      }'
+      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```
Comment (vikkushnir) [inline docker-compose.yml]:
```suggestion
      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
      
```

File: pom.xml [hunk 3]
```diff
@@ -109,6 +109,10 @@
             <artifactId>jjwt-jackson</artifactId>
             <version>0.12.5</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-docker-compose</artifactId>
+        </dependency>
     </dependencies>
```
Comment (vikkushnir) [inline pom.xml]:
```suggestion
```

**Review by vikkushnir — APPROVED** [review]
GJ! Only, better to fix spaces

File: .gitignore (hunk 1 above)
Comment (andrii-hoienko) [reply .gitignore]: Not resolved

File: docker-compose.yml (hunk 2 above)
Comment (andrii-hoienko) [reply docker-compose.yml]: Not resolved

**Review by andrii-hoienko — APPROVED** [review]
Good job!

**Pull request merged**